load_test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2017 ScyllaDB
+
+from avocado import main
+from sdcm.tester import ClusterTester
+
+
+class LoadTest(ClusterTester):
+
+    """
+    Test a Scylla cluster load over a time period.
+
+    :avocado: enable
+    """
+
+    def test_load(self):
+        """
+        Run cassandra-stress with params defined in data_dir/scylla.yaml
```

Comment:
The yaml file is configurable.
 in data_dir/scylla.yaml -> in yaml config

Reply:
Can you describe the content and purpose of this test here? it looks running heavy load (8 loaders) for 1 db node, and persist 7 days.

Reply:
Can we reuse longevity_test with a new config?

Reply:
I don't think so, this test is without nemesis

Reply:
We can make nemesis configurable in longevity_test. If there is a big gap between new loader test and longevity_test. We absolutely need a new file.

Reply:
Load test intended to catch up scylla timeouts/exceptions, currently it does not contain the check itself, and I'm not sure we want to mix it with longevity.

Reply:
Ok